eox_core/edxapp_wrapper/backends/users_h_v1.py

```diff
@@ -197,7 +197,9 @@ def get_enabled_source_methods(cls):
     @staticmethod
     def fetch_from_created_on_site_prop(user, domain):
         """ fetch option """
```

Comment:
Capitalize fetch, please @felipemontoya and add a period at the end of the line.